### PR TITLE
42 write remaining tests

### LINKED ISSUE #42: Write remaining tests
Tests are needed for the following files and their functions:
`routers/cleanup.py`
`routers/filing_cleanup.py`
`routers/institution_cleanup.py`
`repos/submission_repo.py`
`repos/institution_repo.py`
`repos/filing_repo.py`
`config.py`

### PR DESCRIPTION
**Pull request by guffee23**
Closes #42

### CODE REVIEW

File: tests/api/conftest.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        AuthenticatedUser.from_claim(claims),
+    )
+    return auth_mock
+
+
+@pytest.fixture
+def unauthed_user_mock(auth_mock: Mock) -> Mock:
+    auth_mock.return_value = (AuthCredentials("unauthenticated"), UnauthenticatedUser())
+    return auth_mock
+
```
Comment (jcadam14) [inline tests/api/conftest.py:45]: unauthed_user_mock doesn't seem to be used anywhere

File: tests/api/routers/test_institution_cleanup.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    delete_helper_mock = mocker.patch("regtech_cleanup_api.routers.institution_cleanup.delete_helper")
+    delete_helper_mock.return_value = Response(status_code=status.HTTP_204_NO_CONTENT)
+    client = TestClient(app_fixture)
+    res = client.delete("/v1/cleanup/institution/123456E2ETESTBANK123")
+    delete_helper_mock.assert_called_once_with("123456E2ETESTBANK123", ANY)
+    assert res.status_code == 204
+
+
+def test_institution_delete_helper(app_fixture: FastAPI, mocker: MockerFixture, caplog):
+    session_mock = Mock()
```
Comment (jcadam14) [inline tests/api/routers/test_institution_cleanup.py:29]: Learned what caplog is today, nice use here!

File: tests/entities/repos/test_filing_repo.py [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        transaction_session.add(filing3)
+
+        submission1 = SubmissionDAO(
+            id=1,
+            filing=1,
+            submitter_id=2,
+            state=SubmissionState.SUBMISSION_UPLOADED,
+            validation_ruleset_version="v1",
+            submission_time=dt.now(),
+            filename="file1.csv",
```
Comment (jcadam14) [inline tests/entities/repos/test_filing_repo.py:95]: Submissions now have a field called "counter", so make sure you pull in the latest Filing API.  One of the issues with dependencies across repos.  So I believe these DAOs will need to be updated.

File: tests/entities/repos/test_submission_repo.py [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+        submission1 = SubmissionDAO(
+            id=1,
+            filing=1,
+            submitter_id=2,
+            state=SubmissionState.SUBMISSION_UPLOADED,
+            validation_ruleset_version="v1",
+            submission_time=dt.now(),
+            filename="file1.csv",
+        )
```
Comment (jcadam14) [inline tests/entities/repos/test_submission_repo.py:97]: Same with these.

File: tests/entities/repos/test_filing_repo.py (hunk 3 above)
Comment (guffee23) [reply tests/entities/repos/test_filing_repo.py:95]: Will pull the latest and make any changes related to it.

**Pull request merged**